pkg/cloudprovider/aws/suite_test.go

```diff
@@ -207,6 +207,48 @@ var _ = Describe("Allocation", func() {
 				}
 				Expect(nodeNames.Len()).To(Equal(2))
 			})
+			It("should not schedule a non-GPU workload on a node w/GPU", func() {
```

Comment:
Is there any way we can make this a vendor neutral test?

Reply:
Once GPU support itself is vendor neutral, I think this will come along with that.